src/test/java/edu/cornell/kfs/concur/batch/ConcurStandardAccountingExtractToPdpStepTest.java

```diff
@@ -25,26 +25,45 @@
     private static final String DATA_FILE_PATH = "src/test/java/edu/cornell/kfs/concur/batch/fixture/ConcurStandardAccountingExtract.txt";
 
     private ConcurStandardAccountingExtractToPdpStep concurStandardAccountingExtractToPdpStep;
-    private File batchDirectoryFile;
+    //private File batchDirectoryFile;
```

Comment:
If this data element is not needed, shouldn't it be removed instead of commented out?